src/cli/index.js

```diff
@@ -371,3 +371,17 @@ export default function main({
       process.exitCode = 1;
     });
 }
+
+export default function start() {
+  // ignore all arguments after a --
+  const doubleDashIndex = process.argv.findIndex(element => element === '--');
+  const startArgs = process.argv.slice(0, 2);
+  const args = process.argv.slice(2, doubleDashIndex === -1 ? process.argv.length : doubleDashIndex);
+  const endArgs = doubleDashIndex === -1 ? [] : process.argv.slice(doubleDashIndex + 1, process.argv.length);
+
+  main({startArgs, args, endArgs});
+}
+
+if (require.main === module) {
```

Comment:
This runs yarn when you do `node lib/cli`. This part of the code is essentially what makes the single-file builds directly runnable since this file is the entry point.